projects/portal/src/app/views/dialogs/wallets/ununifi/ununifi-import-wallet-with-mnemonic-form-dialog/ununifi-import-wallet-with-mnemonic-form-dialog.component.html

```diff
@@ -48,11 +55,12 @@
               #mnemonicRef
               name="mnemonic"
               [(ngModel)]="privateWallet.mnemonic"
-              (input)="onChangeIdAndMnemonic(idRef.value, mnemonicRef.value)"
+              (change)="onChangeIdAndMnemonic(idRef.value, mnemonicRef.value)"
```

Comment:
(before) Trim every time you enter a character.
(after) Trim after the cursor OFF. 
*This is same as walletID imput

Reply:
Using the chanage event, if a user try to press the button immediately after copying his/her mnemonic, the private key is not in the input box and he/she cannot press the button to proceed.

I concern this may be confuse the user.

Reply:
I see, certainly It need to put cursor on other place. 
I will change this fix by using input event.